abs.yaml

```diff
@@ -0,0 +1,4 @@
+aggregate_check: false
```

Comment:
```suggestion
```
this was for when I did a PR against your branch

Reply:
will remove.